src/scanners/css.js

```diff
@@ -26,7 +26,15 @@ export default class CSSScanner extends BaseScanner {
                 }
 
                 this.validatorMessages = this.validatorMessages.concat(
-                  rules[cssRule](rule));
```

Comment:
Again here - it gets confusing - (totally my fault) but it'll make sense to clean this up now. We need to differentiate between cssRules as in bits of CSS and the rule functions. To be consistent with the previous suggestion we need to make `cssRule` -> `cssRuleFunc` and `rule` -> `cssRule` I think.